app/models/reservation_scopes.rb

```diff
@@ -1,51 +1,37 @@
 module ReservationScopes
   def self.included(base) # rubocop:disable MethodLength, AbcSize
     base.class_eval do
-      scope :recent, ->() { order('start_date, due_date, reserver_id') }
+      default_scope { order('start_date, due_date, reserver_id') }
       scope :user_sort, ->() { order('reserver_id') }
-      scope :finalized, lambda {
```

Comment:
Here's a thought: isn't this basically the same as `where.not(status: [:requested, :denied])`?